airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -15,7 +15,11 @@ acceptance_tests:
   discovery:
     tests:
     - config_path: "secrets/config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.4.6"
```

Comment:
@grubberr This is because of removing "repository" from the `teams` stream, which was never outputting data, is that correct? Anything else that flagged as non-compatible?

Reply:
@erohmensing  I have wrote all possible compatibility problems in PR descriptions

Reply:
as for me we need to improve compatibility tests some of them are definitely false positive

Reply:
I think, looking at the error message, that it's only[ this specific one](https://github.com/airbytehq/airbyte/actions/runs/4546946491/jobs/8016229228#step:11:1595) that is incompatible, though I agree the logging of the diff is quite confusing. It would make sense as to why the backwards compatibility test didn't start failing until after you removed that repository param, despite having added a lot of params.